Tests iOS/Support/Elements/TabBarElement.swift

```diff
@@ -18,4 +18,8 @@ struct TabBarElement: PocketUIElement {
     var accountButton: XCUIElement {
         element.buttons["Account"]
     }
+    
+    var settingsButton: XCUIElement {
+        element/*@START_MENU_TOKEN@*/.buttons["account-tab-bar-button"]/*[[".buttons[\"Settings\"]",".buttons[\"account-tab-bar-button\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/
```

Comment:
ohh interesting didn't realize these comments appear - was this generated by the recorder?

Reply:
Recorder generated that yep, I'll remove those - didn't know that would be pushed up as text like that